pkg/client/observatorium/observability_config.go

```diff
@@ -123,6 +124,12 @@ type ObservabilityCloudwatchLoggingConfigCredentials struct {
 	SecretAccessKey string `yaml:"aws_secret_access_key" validate:"required"`
 }
 
+type ObservabilityEnterpriseCloudwatchLoggingConfigCredentials struct {
+	AccessKey       string `yaml:"aws_access_key" validate:"-"`
+	SecretAccessKey string `yaml:"aws_secret_access_key" validate:"-"`
+	OrgId           string `yaml:"org_id" validate:"-"`
```

Comment:
Accronyms in Go are uppercased as long as the don't affect visibility of the attribute. In this case this should be named `OrgID

Reply:
:+1: